task-2/dist/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
You also need to exclude the `dist` directory from the checks of Prettier the same way you did with the ESLint checks.

Reply:
.prettierignore file has been added which excludes the prettier checks for the dist directory.